pkg/generate/app/cmd/newapp.go

```diff
@@ -312,17 +314,19 @@ func (c *AppConfig) componentsForRepos(repositories app.SourceRepositories) (app
 			errs = append(errs, fmt.Errorf("source not detected for repository %q", repo))
 			continue
 		case info.Dockerfile != nil && (len(c.Strategy) == 0 || c.Strategy == "docker"):
-			df, err := info.Dockerfile.Dockerfile()
-			if err != nil {
-				// an unparseable docker file should not terminate the entire create - it may be valid to Docker but not to us
```

Comment:
We should stop right away if the Dockerfile is unparseable: no chance Docker will be able to build it, since we rely on Docker's Dockerfile parser.